diff --git a/lib/cc/analyzer/engine.rb b/lib/cc/analyzer/engine.rb
index 000e943..575e6ab 100644
--- a/lib/cc/analyzer/engine.rb
+++ b/lib/cc/analyzer/engine.rb
@@ -74,7 +74,6 @@ module CC
           "--memory", memory_limit,
           "--memory-swap", "-1",
           "--net", "none",
-          "--rm",
           "--volume", "#{@code_path}:/code:ro",
           "--volume", "#{config_file.host_path}:/config.json:ro",
           "--user", "9000:9000"
